### PR TITLE
ci: add bench workflow

### LINKED ISSUE #45: Add a benchmarks workflow
It might be useful to get a basic view on the performance impact of the PRs.

### PR DESCRIPTION
**Pull request by dnlup**
Closes #45

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
         needs: [test]
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/setup-node@v2.4.1
+            - uses: actions/setup-node@v2
```
Comment (simonecorsi) [inline .github/workflows/release.yml:33]: Keep in mind that dependant bot pins the version

**Comment by simonecorsi** [comment]
Would be helpful instead checking the bench with lint-staged and outputting to a file so when we get MRs we can clearly see the diff?

**Review by simonecorsi — APPROVED** [review]

**Comment by antoniomuso** [comment]
I agree with Simone; We can do the benchmark and then do a diff with the result of the target branch. WDYT?

**Review by antoniomuso — APPROVED** [review]

**Comment by simonecorsi** [comment]
I'll merge this meanwhile, we can discuss improvements on a new issue if we want to change it

**Pull request merged**